### ISSUE TITLE
deployOnSave does not work

### ISSUE THREAD

**Issue by azielke** [mkloubert/vscode-deploy-reloaded#16]
### Description

I have "deployOnSave": true in my Package Configuration, but files are not deployed when saved.

### Actual behavior

nothing happens

### Expected behavior

file copied to target

### Example config

```json
{
    "deploy.reloaded": {
        "showPopupOnSuccess": false,
        "packages": [
            {
                "name": "production",
                "deployOnSave": true,

                "exclude": [
                    ".vscode/**",
                    "*.code-workspace"
                ],

                "targets": [ "db1" ]
            }
        ],
        "targets": [
            {
                "type": "sftp",
                "name": "db1",
                "host": "10.10.10.1", "port": 22,
                "user": "admin", "password": "password",

                "dir": "/var/www",
            }
        ]
    }
}
```

### Your environment

- Operating system: Windows 10
- Visual Studio Code version: 
```Version 1.19.2
Commit 490ef761b76b3f3b3832eff7a588aac891e5fe80
Date 2018-01-10T15:55:03.538Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
```
- Extension version: 0.32.0

### Additional comments

When I add "fastCheckOnSave": true to the package config, deployOnSave works again.

**Comment by mkloubert** [comment]
@azielke 

**Should be fixed since 0.32.1.**

**Comment by azielke** [comment]
Installed 0.32.1 - can confirm it works even without "fastCheckOnSave". Thanks a lot!
